### ISSUE TITLE
Ver el curso de deep learning CS231n de Stanford

### ISSUE THREAD

**Issue by giorg0s** [giorg0s/2020-tfg-jorge_guerrero#1]
El objetivo es profundizar el conocimiento acerca del concepto de redes neuronales, concretamente las redes neuronales convolucionales (CNNs) usando para ello algunas clases impartidas en el curso CS231n de la _Stanford University School of Engineering_.

**Comment by giorg0s** [comment]
- [x] [Lecture 1. Introduction to Convolutional Neural Networks for Visual Recognition](https://www.youtube.com/watch?v=vT1JzLTH4G4&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=1) 

- [x] [Lecture 2. Image Classification](https://www.youtube.com/watch?v=OoUX-nOEjG0&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=2)

- [x] [Lecture 3. Loss Functions and Optimization](https://www.youtube.com/watch?v=h7iBpEHGVNc&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=3)

- [x] [Lecture 4. Introduction to Neural Networks]( https://www.youtube.com/watch?v=d14TUNcbn1k&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=4)

- [x] [Lecture 5. Convolutional Neural Networks](https://www.youtube.com/watch?v=bNb2fEVKeEo&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=5)

- [x] [Lecture 6. Training Neural Networks I](https://www.youtube.com/watch?v=wEoyxE0GP2M&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=6)

- [x] [Lecture 7. Training Neural Networks II](https://www.youtube.com/watch?v=_JB0AO7QxSA&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=7)

- [x] [Lecture 8. Deep Learning Software](https://www.youtube.com/watch?v=6SlgtELqOWc&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=8)

- [x] [Lecture 9. CNN Architectures](https://www.youtube.com/watch?v=DAOcjicFr1Y&list=PLC1qU-LWwrF64f4QKQT-Vg5Wr4qEE1Zxk&index=9)

**Comment by giorg0s** [comment]
Esta herramienta es muy interesante porque se pueden ver CNNs en funcionamiento a través del navegador:
https://cs.stanford.edu/people/karpathy/convnetjs/
